tests/init.php

```diff
@@ -34,33 +34,25 @@
 use Cake\Log\Log;
 use Cake\Routing\Router;
 
-Configure::write(
-    'App',
-    [
+Configure::write('debug', true);
+Configure::write('App', [
     'namespace' => 'App',
     'paths' => [
         'plugins' => [APP . DS . 'testapp' . DS . 'Plugin' . DS],
     ],
-    ]
-);
+]);
 
-Cache::setConfig(
-    '_cake_core_',
-    [
+Cache::setConfig('_cake_core_', [
     'className' => 'File',
     'path' => sys_get_temp_dir(),
-    ]
-);
+]);
 
-Log::setConfig(
-    [
-    'debug' => [
-        'engine' => 'Cake\Log\Engine\FileLog',
-        'levels' => ['notice', 'info', 'debug'],
-        'file' => 'debug',
-    ],
-    ]
-);
+Log::setConfig('debug', [
+    'engine' => 'Cake\Log\Engine\FileLog',
```

Comment:
```suggestion
    'engine' => FileLog::class,
```

This might need to be declared.